libs/community/langchain_community/llms/huggingface_pipeline.py

```diff
@@ -11,7 +11,12 @@
 
 DEFAULT_MODEL_ID = "gpt2"
 DEFAULT_TASK = "text-generation"
-VALID_TASKS = ("text2text-generation", "text-generation", "summarization")
+VALID_TASKS = (
+    "text2text-generation",
+    "text-generation",
+    "summarization",
+    "translation",
```

Comment:
Guess it could benefit from a constant variable `TRANSLATION_TASK` that could be shared 🤔